src/renderer/component/publishForm/internal/bid-help-text.jsx

```diff
@@ -58,16 +59,35 @@ class BidHelpText extends React.PureComponent<Props> {
         </React.Fragment>
       );
     }
-
-    return winningBidForClaimUri ? (
+    
+    if (!winningBidForClaimUri && !winndingBidForClaimWithChannel) {
+      return __('Any amount will give you the winning bid.');
+    }
+    
+    const generateTakeOverMessage = (amount, uri) => {
+      return `${__('A deposit greater than')} ${amount} ${__('is needed to win')} ${uri}`
+    }
+    
+    let content;
+    if (winndingBidForClaimWithChannel) {
+      content = `Winning bid is ${winndingBidForClaimWithChannel}`
+    }
+    
+    // <div>
+    //   {__('However, you can still get')}{' '}
+    //   {(winndingBidForClaimWithChannel && winningBidForClaimUri) ? __('these URLs for any amount')}
```

Comment:
I'm assuming you'll add this back in :)

Reply:
Yes. This PR still needs a lot of cleanup.